core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -184,6 +184,8 @@ message ListStatusPOptions {
   optional LoadMetadataPType loadMetadataType = 2;
   optional FileSystemMasterCommonPOptions commonOptions = 3;
   optional bool recursive = 4;
+  // Only loads metadata about a path in the UFS to Alluxio. No data will be transferred.
+  optional bool loadMetadataOnly = 5;
```

Comment:
How does this option behave if the other option `LoadMetadataPType loadMetadataType` is set to `NEVER`? It looks like this option is really about sending results back or not, not about loading metadata? Is this a correct understanding?

If this option only controls returning results to the client or not, then we should rename the option to be more specific about returning results, instead of mentioning loading metadata.

Reply:
the understanding is correct.
perhaps renamed this flag to something like `resultsRequired`? @gpang

Reply:
the flag added to `ListStatusPOption`